### ISSUE TITLE
About the ensemble model

### ISSUE THREAD

**Issue by tuji-sjp** [jacobgil/pytorch-grad-cam#38]
I fused the logits of the three models. How does the ensemble model get GradCAM?

**Comment by jacobgil** [comment]
Interesting!
I think for Grad-CAM this will be equivalent to getting 3 (unormalized) heatmaps separately and then fusing them.
